sklearn/utils/_array_api.py

```diff
@@ -0,0 +1,124 @@
+"""Tools to support array_api."""
+import numpy
+from .._config import get_config
+import scipy.special as special
+
+
+class _ArrayAPIWrapper:
+    def __init__(self, array_namespace):
+        self._namespace = array_namespace
+
+    def __getattr__(self, name):
+        return getattr(self._namespace, name)
+
+    def astype(self, x, dtype, *, copy=True, casting="unsafe"):
+        # support casting for NumPy
+        if self._namespace.__name__ == "numpy.array_api":
+            x_np = numpy.asarray(x).astype(dtype, casting=casting, copy=copy)
+            return self._namespace.asarray(x_np)
+
+        f = self._namespace.astype
+        return f(x, dtype, copy=copy)
+
+    def asarray(self, obj, *, dtype=None, device=None, copy=None, order=None):
+        f = self._namespace.asarray
+
+        # support order in NumPy
```

Comment:
```suggestion
        # Extend Array API to support `order` in NumPy
```

Reply:
Same question here: any issue or discussion to link to avoid having to maintain this extension of the standard in the long term?

Reply:
The discussion around `order` is here: https://github.com/data-apis/array-api/issues/122